src/frontend/komponenter/Fagsak/Personlinje/Behandlingsmeny/LeggTilEllerFjernBrevmottakere/LeggTilEllerFjernBrevmottakere.tsx

```diff
@@ -97,7 +97,7 @@ const LeggTilEllerFjernBrevmottakere: React.FC<IProps> = ({ åpenBehandling, erL
                         fullmektig, verge eller dødsbo.
                     </StyledAlert>
                     {åpenBehandling.brevmottakere.map(mottaker => (
-                        <BrevmottakerTabell mottaker={mottaker} />
+                        <BrevmottakerTabell mottaker={mottaker} key={`mottaker-${mottaker.id}`} />
```

Comment:
Retter en warning i konsollen fra react angående nøkler på mappede elementer https://beta.reactjs.org/learn/rendering-lists#keeping-list-items-in-order-with-key